python/adjoint/objective.py

```diff
@@ -284,11 +293,30 @@ def __call__(self):
 
 
 class FourierFields(ObjectiveQuantity):
+    """A differentiable frequency-dependent Fourier Fields (dft_fields)
+    Attributes:
```

Comment:
Replace "Attributes:" with "Args:" and separate it with the one-line summary above it with a line break following the style guide: https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods.

Reply:
`Attributes` is used for classes and `Args` for functions.

For the `Attributes` section of the docstrings for the `EigenmodeCoefficient` class, we would need to list its *public* members, almost anything with a `self.` prefix defined in the constructor excluding `self._` which are generally used for private members.

We should also add a separate `Args` section to the constructor. The argument descriptions in the constructor docstrings should hopefully be different than the attributes description in the class docstring. The attributes description should be a general summary whereas the argument description is less so.

Reply:
I added separate ``Args`` sections to the constructors. I tried to make the attribute description more concise and general, and argument description more detailed and elaborated. But they still seem a little redundant to me.